client.js

```diff
@@ -49,6 +21,18 @@ bot.on('ready', () => {
     logger.info(`Logged in as ${bot.user.tag}!`);
 });
 
+//invite in case I can't find server after leaving https://discord.gg/Q8nzh5
+
+//server greeting for users who join
+bot.on('guildMemberAdd', (member) => {
+    let guild = member.guild; // Reading property `guild` of guildmember object.
+    let memberTag = member.user.tag; // GuildMembers don't have a tag property, read property user of guildmember to get the user object from it
+    if (guild.systemChannel){ // Checking if it's not null
+        guild.systemChannel.send(`Welcome to the server, ${memberTag}!`);
```

Comment:
So, what exactly IS our goal here? If it's really just to say "hi!" I don't know if this is really worth it. We might want to have it PM them off the bat, maybe with the helptext and a quick welcome blurb, so they can have some info about how things work?